autoapi/mappers/python.py

```diff
@@ -36,10 +36,14 @@ def load(self, patterns, dirs, ignore=None):
         shortened, relative path the package/module
         """
         for dir_ in dirs:
+            dir_root = dir_
+            if os.path.exists(os.path.join(dir_, '__init__.py')):
```

Comment:
We can remove this `if` and make the following line the default behaviour.
Please could you add a test for a package with an implicit namespace as well.